src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -431,6 +449,10 @@ public interface IStartConfiguration
             /// No update checks or analytics collection should be done.
             /// </summary>
             bool IsHeadless { get; set; }
+            /// <summary>
+            /// Host analytics info
+            /// </summary>
+            HostAnalyticsInfo HostAnalyticsInfo { get; set; }
```

Comment:
This is now part of `IStartConfiguration` so integrators can set it there

Reply:
This is a breaking change - you cannot change a public interface like this - it means that anyone that implemented the interface will no longer conform to it. Instead add the new property to the concrete class DefaultStartConfiguration and if you prefer you can add a new interface.

Reply:
Thanks @mjkkirschner . Addressed